components/MenuItem/index.js

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+import {withTheme} from 'styled-components';
+
+import IcArrow from '../../icons/IcArrowForward'
```

Comment:
Podes usar aqui o import relativo. Essa pasta tem um index.

```suggestion
import {IcArrow} from '../../icons'
```

Reply:
Cara, o problema que tá dando é que se eu exporto só o icone, o componente não carrega no storybook, fica tudo branco, podemos revisar.